### PR TITLE
Feat/Add notification object attributes

### PR DESCRIPTION
**Pull request by carpawell**
Related to #193.

I think it should not be included in the next (`2.11.0`+) release, so it may wait.

### CODE REVIEW

File: object/types.proto [hunk 1]
```diff
@@ -117,6 +117,12 @@ message Header {
   //   Marks smaller parts of a split bigger object
   // * __NEOFS__EXPIRATION_EPOCH \
   //   Tells GC to delete object after that epoch
+  // * __NEOFS__TICK_TOPIC \
+  //   UTF-8 string topic ID that is used for object notification
+  // * __NEOFS__TICK_EPOCH \
+  //   Base10 encoded number that defines what epoch must produce
```
Comment (realloc) [inline object/types.proto:123]: Base10 -> decimal

File: object/types.proto [hunk 2]
```diff
@@ -117,6 +117,12 @@ message Header {
   //   Marks smaller parts of a split bigger object
   // * __NEOFS__EXPIRATION_EPOCH \
   //   Tells GC to delete object after that epoch
+  // * __NEOFS__TICK_TOPIC \
```
Comment (realloc) [inline object/types.proto:120]: It' better to put it after `TICK_EPOCH` attribute, to simplify understanding.

File: object/types.proto [hunk 3]
```diff
@@ -117,6 +117,12 @@ message Header {
...
+  //   UTF-8 string topic ID that is used for object notification
+  // * __NEOFS__TICK_EPOCH \
+  //   Base10 encoded number that defines what epoch must produce
+  //   object notification (`0` value produces notification right
+  //   after object put)
```
Comment (realloc) [inline object/types.proto:125]: I'd also add a note that the notification body will contain object address in UFT-8 string.

**Review by realloc — CHANGES_REQUESTED** [review]

File: object/types.proto (hunk 3 above)
Comment (carpawell) [reply object/types.proto:125]: thought such details should be in spec, but added

Comment (realloc) [reply object/types.proto]: NeoFS Spec is automatically generated from this API definitions. =)

Comment (carpawell) [reply object/types.proto]: i meant a few manually written words about notifications

**Review by cthulhu-rider — APPROVED** [review]

**Review by fyrchik — APPROVED** [review]

**Review by realloc — APPROVED** [review]

**Pull request merged**